src/logging/NewRelicLoggingService.js

```diff
@@ -137,13 +146,13 @@ export default class NewRelicLoggingService {
         Ignored errors are logged via adding a page action.
         Other errors are logged via noticeError and count as "JS Errors" for the application.
     */
-    const errorMessage = typeof error === 'string' ? error : error.message;
+    const errorMessage = errorObject.message || (typeof errorObject === 'string' ? errorObject : '');
```

Comment:
Given this check, is it fair to say that logError also effectively takes an `infoStringOrErrObject`?

Reply:
Yes - I even added a test so it's tested both ways. I was attempting to discourage an error string - but the code should either support it or not. I'll change the name to `errorStringOrObject`.